app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      user = User.find(params[:id])
```

Comment:
Check how cancancan works. https://github.com/CanCanCommunity/cancancan/wiki/Authorizing-controller-actions
This is already done with **load_and_authorize_resource**